spec/models/donation_site_spec.rb

```diff
@@ -24,6 +27,18 @@
     it "is invalid without an address" do
       expect(build(:donation_site, address: nil)).not_to be_valid
     end
+
+    it "is valid without an contact name" do
+      expect(build(:donation_site, contact_name: nil)).to be_valid
+    end
+
+    it "is valid without an phone" do
+      expect(build(:donation_site, phone: nil)).to be_valid
+    end
+
+    it "is valid without an email" do
```

Comment:
Email and contact name have some other validations. It would be good to test a valid email and a valid contact

Reply:
@doliveirakn yes! I added those now and also a test for phone numbers. The DoD of the issue was not clear about it. But after asking, all 3 attributes added to this entity should be optional. Next time I will edit the issue to make it clearer, since we are working together.